apps/docs/content/components/Grid/index.mdx

```diff
@@ -127,175 +127,49 @@ We recommend using either the grid or stack component, depending on how the chil
 
 ### Grid examples
 
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <Text as="h4">Page or section layout</Text>
-    <Text as="p" mt="0">
-      When defining the page or a section layout. E.g. A page with sidebar, main
-      and navigation areas.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/7c418d9e-3e44-4a0e-9f4f-18f5a62d89d0"
-    role="presentation"
-    width="456"
-  />
-</Container>
-
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <Text as="h4">Sub sections</Text>
-    <Text as="p" mt="0">
-      When dividing an area into two areas. E.g. A "Send feedback" sidebar and a
-      form, and the form uses the grid for the form layout itself.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/d617bfea-0758-4066-9ccc-7853eeb35cf3"
-    role="presentation"
-    width="456"
-  />
-</Container>
-
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <h4 style={{marginBottom: 0}}>Card or pillar grids</h4>
-    <Text as="p" mt="0">
-      When laying out cards, pillars, or similar content-based elements.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/56dfbcd8-72ff-401c-a530-25f11f671f7f"
-    role="presentation"
-    width="456"
-  />
-</Container>
+#### Page or section layout
+
+When defining the page or a section layout. E.g. A page with sidebar, main and navigation areas.
+
+![](https://github.com/primer/brand/assets/912236/7c418d9e-3e44-4a0e-9f4f-18f5a62d89d0)
```

Comment:
I like this but it does feel quite product-oriented., and we don't have many helper components to enforce it like PageLayout. Should this focus on footer instead of a sidebar? Not a strong opinion either way though.

Reply:
Good point, I changed the description and the image to remove the sidebar and use a footer area example